### PR TITLE
Enable python code style checks in whole test directory

### PR DESCRIPTION
**Pull request by KKoukiou**
Since we are upstreaming the selenium tests for machines I thought it is good timing to enable pep8 for whole test directory.

### CODE REVIEW

File: test/avocado/selenium-base.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import os
 import sys
 machine_test_dir = os.path.dirname(os.path.abspath(__file__))
-if not machine_test_dir in sys.path:
+if machine_test_dir not in sys.path:
```
Comment (martinpitt) [inline test/avocado/selenium-base.py]: Commit message: it's "remainder" :-)

**Comment by martinpitt** [comment]
@KKoukiou : Did you deliberately not trigger tests here? (But I don't see a [no-test]). Or is that infra failure again, perhaps during restarting the webhook? (CC @Gundersanne)

File: test/common/testlib.py [hunk 2]
```diff
@@ -1417,7 +1417,7 @@ def wait(func, msg=None, delay=1, tries=60):
             val = func()
             if val:
                 return val
-        except:
+        except Error:
```
Comment (martinpitt) [inline test/common/testlib.py]: I think this is why so many tests fail -- we also often use this function to wait for a `Machine.execute()` to succeed. Thus the tests now fail on `subprocess.CalledProcessError`. Maybe try with `(Error, subprocess.SubprocessError)` for now, and if that still fails, we need to generalize it further?

**Comment by KKoukiou** [comment]
> @KKoukiou : Did you deliberately not trigger tests here? (But I don't see a [no-test]). Or is that infra failure again, perhaps during restarting the webhook? (CC @Gundersanne)

I didn't deliberately stop tests from being executed.

**Comment by martinpitt** [comment]
Bah, needs [TimeoutException](https://fedorapeople.org/groups/cockpit/logs/pull-11199-20190219-083535-57a865d5-selenium-chrome/log.html#6) as well. We really designed this badly, so for now maybe let's bite the bullet and watch for `Exception`? (It's *really* bad to do this, but then again it's just tests).

**Pull request merged**